Frontend/ui-customer/src/Services/Accessor.purs

```diff
@@ -250,3 +250,6 @@ _currentStatus = lens (unwrap >>> _.currentStatus) (\oldRec newVal -> wrap ((unw
 
 _fareBreakup :: forall a b c. Newtype a { fareBreakup :: b | c } => Lens' a b
 _fareBreakup = lens (unwrap >>> _.fareBreakup) (\oldRec newVal -> wrap ((unwrap oldRec) { fareBreakup = newVal }))
+
+_payment_method :: forall a b c. Newtype a { payment_method :: b | c } => Lens' a b
```

Comment:
follow camelCase here - `_parentMethod`

Reply:
This payment method is coming for global payload. There we are following snake case.

Reply:
yeah variable can be payment_method and it's okay to keep it that way but the accessor functions are having a common patter where parentMethod can follow it. So it would look like - 

`_paymentMethod :: forall a b c. Newtype a { payment_method :: b | c } => Lens' a b`